crates/ef-testing/src/models/case.rs

```diff
@@ -168,7 +164,7 @@ impl BlockchainTestCase {
         let kakarot_address = kakarot.kakarot_address;
 
         // Get lock on the Starknet sequencer
-        let starknet = env.sequencer().sequencer.backend.state.read().await;
+        let mut starknet = env.sequencer().sequencer.backend.state.write().await;
```

Comment:
is there a reason why we are calling `write` here?

Reply:
See the above comment on the Elias remark.